### DISCUSSION TITLE
About the pre or next page

### DISCUSSION THREAD

**Discussion by SWHL** [colinwilson/lotusdocs#91] [Q&A] [answered]
I am a little confused that which parameter controls the pre or next page in the bottom of page.
![image](https://github.com/colinwilson/lotusdocs/assets/28639377/fee0792a-3a5d-4c96-9294-237211c3adf4)

I have tried the `date` and `lastmod` in the header of md file. But it is not always useful.

Meawhile, I find nothing about the parameter in the [docs](https://lotusdocs.dev/docs/).

Appreciate any help on this.

**Comment by Nadine2016** [comment, marked as answer]
@SWHL you need to use the weight attribute in the front matter. it is a little tricky as your weight must be aligned across all files.

**Comment by colinwilson** [comment]
@SWHL, @Nadine2016 is correct.

> it is a little tricky as your weight must be aligned across all files.

I am testing an optional weight indicator feature that can be configured in the `hugo.toml` config file. It'll be present only when working locally and shows the weight of each page, nav link and menu item. Should help with organising content.

**Reply by SWHL** [reply]
Thanks, this solved my problem.
